src/System.Runtime.WindowsRuntime/src/System/Resources/WindowsRuntimeResourceManager.cs

```diff
@@ -231,7 +231,20 @@ private static void GlobalResourceContextChanged(object sender, IMapChangedEvent
             Debug.Assert(s_globalResourceContextFallBackList != null);
             Debug.Assert(s_globalResourceContext != null);
 
-            List<String> languages = new List<string>(s_globalResourceContext.Languages);
+            IReadOnlyList<string> langs;
+
+            try
+            {
+                langs = s_globalResourceContext.Languages;
+            }
+            catch (ArgumentException)
+            {
+                // Sometimes Windows Runtime fails and we get Argument Exception which can fail fast the whole app
+                // to avoid that we ignore the exception.
```

Comment:
Go ahead.  Personally I'd prefer in situations like this if we had an issue it was tied back to (catching ArgumentExceptions in production code is generally an indication that something else is wrong), but if you're confident this code is going to remain, that's fine.